### PR TITLE
fix(TDP-3061): put actions label in sentence case

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3061

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [x] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

File: dataprep-backend-service/src/main/resources/org/talend/dataprep/messages.properties [hunk 1]
```diff
@@ -535,7 +535,7 @@ parameter.lookup_ds_name.desc=Lookup dataset name
 parameter.lookup_ds_id.label=Lookup dataset id
 parameter.lookup_ds_id.desc=Lookup dataset id
 
-parameter.lookup_ds_url.label=Lookup dataset url
+parameter.lookup_ds_url.label=Lookup dataset URL
 parameter.lookup_ds_url.desc=Lookup dataset url
```
Comment (frassinier) [inline dataprep-backend-service/src/main/resources/org/talend/dataprep/messages.properties]: If you change case of the label, perhaps you should consider its description too?

File: dataprep-backend-common/src/main/resources/org/talend/dataprep/i18n/actions_messages.properties [hunk 2]
```diff
@@ -560,7 +560,7 @@ parameter.lookup_ds_name.desc=Lookup dataset name
 parameter.lookup_ds_id.label=Lookup dataset id
 parameter.lookup_ds_id.desc=Lookup dataset id
 
-parameter.lookup_ds_url.label=Lookup dataset url
+parameter.lookup_ds_url.label=Lookup dataset URL
 parameter.lookup_ds_url.desc=Lookup dataset url
```
Comment (frassinier) [inline dataprep-backend-common/src/main/resources/org/talend/dataprep/i18n/actions_messages.properties]: If you change case of the label, perhaps you should consider its description too?

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/825/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/825/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/828/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/828/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/829/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/829/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/830/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/830/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/836/) [:star:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/836/)

**Pull request merged**